apps/aecore/src/aec_pinning_agent.erl

```diff
@@ -0,0 +1,167 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2024, Aeternity
+%%% @doc
+%%% Gather cild->parent chain pinning information for AE HC and make it 
+%%% available through API
+%%% @end
+%%%-------------------------------------------------------------------
+
+-module(aec_pinning_agent).
+-define(PIN_TAG, 101).
+
+-export(
+    [
+    get_pinning_data/0,
+    encode_pin_payload/1,
+    decode_pin_payload/1,
+    get_pins/3,
+    create_pin_tx/6,
+    post_pin_tx/2
+    ]).
+
+
+get_pinning_data() ->
+    {ok, #{
+        epoch := Epoch,
+        first := First,
+        last := _Last,
+        length := _Length
+    }} = aec_chain_hc:epoch_info(),
+    %BlockHash = aec_chain:get_block_hash_optionally_by_hash_or_height(First-1),
+    {ok, BlockHash} = aec_chain_state:get_key_block_hash_at_height(First-1),
+    ConMod = aec_parent_connector:get_parent_conn_mod(),
+    {_,Type} = lists:split(8,atom_to_list(ConMod)), % split off "aehttpc_" from mod name to get type
```

Comment:
Indeed we could, we could considering also remove the type altogether (until further notice) since we won't implement btc/doge for this sprint likely. Let's keep this as is now, since we'll be returning to the code for further work ASA the PR gets merged.